src/test/java/seedu/address/logic/parser/EntryBookListParserTest.java

```diff
@@ -108,6 +110,20 @@ public void parseCommand_feed() throws Exception {
         assertTrue(parser.parseCommand(FeedCommand.COMMAND_WORD + " https://url.com/a.xml") instanceof FeedCommand);
     }
 
+    @Test
+    public void parseCommand_bing() throws Exception {
+        assertTrue(parser.parseCommand(BingWebSearchCommand.COMMAND_WORD + " somekeyword")
+                instanceof BingWebSearchCommand);
+    }
+
+    @Test
+    public void parseCommand_googlenews() throws Exception {
+        assertTrue(parser.parseCommand(GoogleNewsCommand.COMMAND_WORD) instanceof GoogleNewsCommand);
+        assertTrue(parser.parseCommand(GoogleNewsCommand.COMMAND_ALIAS) instanceof GoogleNewsCommand);
+        assertTrue(parser.parseCommand(GoogleNewsCommand.COMMAND_WORD + " kw") instanceof GoogleNewsCommand);
```

Comment:
Should we instead do `assertEquals(parser.parseCommand(GoogleNewsCommand.COMMAND_WORD + "kw"), new GoogleNewsCommand("kw"))`?

Reply:
Also, because `add` is promoted to all contexts, I want to make the following note:

Right now `addListEntry#ModelManager` (called with an `add` command) calls `updateFilteredList#ModelManager`. This behavior will be removed in #175.

The `find` command is restricted to the list context right now, but it should be able to be promoted to all contexts without breaking anything (added to archives context in #175 and doesn't appear to break anything).

Reply:
To remain consistent with the previous tests in here I decided to just assert the instanceof. This will also make life easier if we ever add new options to `gnews` command.

----
The behaviour for when the predicate resets isn't even documented so anything that works will be fine, we just have to be consistent. (If `add` in list context resets predicate so should `sub` in feeds context for example.)